module/VuFind/src/VuFind/Auth/OpenIDConnect.php

```diff
@@ -0,0 +1,270 @@
+<?php
+/**
+ * OpenID Connect authentication module.
+ *
+ * PHP version 7
```

Comment:
If we move forward with this, we'll need to update this to PHP version 8 now that minimum requirements have changed.